src/onegov/activity/iso20022.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, **kwargs):
         for key, value in kwargs.items():
             setattr(self, key, value)
 
-        self.username = None
+        self.username = ''  # Has to be str for comparison used in sort
```

Comment:
This was the root cause of the infamous error we were getting a lot: 
`TypeError: '<' not supported between instances of 'NoneType' and 'str'`